About.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Buganda Spirit</title>
+</head>
+<body>
+    <div>
+        <div class="top-mobile">
+
+        </div>
+    <div class="topcontact">
+        <ul>
+            <li><img src="https://img.icons8.com/ios-glyphs/18/ffffff/facebook-f.png"/></li>
+            <li><img src="https://img.icons8.com/ios-glyphs/18/ffffff/twitter--v1.png"/></li>
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image